docs/guide/acquiring_data/database.rst

```diff
@@ -315,7 +315,7 @@ files are downloaded.
     ...     vso.attrs.Time('2012-08-05 00:00:00', '2012-08-05 00:00:01'),
     ...     vso.attrs.Instrument('AIA'))  # doctest: +REMOTE_DATA
     >>> len(database)  # doctest: +SKIP
-    37
+    70
```

Comment:
Notice that on the second paragraph below, it makes reference to the old value (37). Please, update it with the new on.